### PR TITLE
Fix typo in methods.js

### PR DESCRIPTION
**Pull request by eltociear**
inital -> initial

### CODE REVIEW

**Review by haltman-at — APPROVED** [review]

**Review by dongmingh — APPROVED** [review]
The change is good.

**Comment by haltman-at** [comment]
(Note: We're currently in the middle of a release, please wait till after to merge this!)

**Pull request merged**
